manifests/a/Adobe/AcrobatProDC/2022.0.0/Adobe.AcrobatProDC.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Adobe.AcrobatProDC
+PackageVersion: 2022.0.0
+InstallerType: zip
+NestedInstallerType: exe
+NestedInstallerFiles:
+- RelativeFilePath: Setup.exe
+InstallerSwitches:
+  Silent: /sALL /rs /rps /msi EULA_ACCEPT=YES

```

Comment:
Because you have a EULA agreement in the switches, you must provide an `Agreements` section with the EULA and corresponding URL to the EULA in the locale manifest